lennykamande/andela_bootcamp_learning
# Ft website design dashboard 2110701

What does this PR do?
Have a template of the website look of the Shopping cart that we will use

Description of Task to be completed?
Have the following

Project UML diagram
Project Wireframe
Designs Folder(Dashboard, login, Register html templates/ CSS files)
How should this be manually tested?
Download the project,
Go to the Designs folder
Click on each template and see the resource created.

What are the relevant pivotal tracker stories?
#2110701
